content/participants/ttjc@gunishmatta/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Gunish Matta
+type: "participant"
+name: "Gunish Matta"
+portfolio: ""
+github-link: "https://github.com/gunishmatta"
+linkedin-link: "https://linkedin.com/in/gunishmatta2"
+looking-for: "job"
+description: "Competitive skills in  HTML,CSS,Javascript and React"
+---
+
+Hi! I am Gunish Matta, let me walk you through my intro, projects and blogs.
+
+### introduction
+I am Gunish Matta, a Computer Science Graduate, fond of learning latest things in Web Development and skilled in various technologies like HTML,CSS,Javascript,Node and React.
+
+## why are you taking this challenge?
+
+I need a job to start my professional career.
+
+## something interesting about you?
+
+I love to learn new things.
+
+### help I need?
+
+It would be good if someone tells me how to do build complex projects with React and Redux Saga and a little bit advanced Backend.
+
+### how to contact me?
+
+- Twitter
```

Comment:
Hey! @gunishmatta you need to add links to these platforms too in markdown format like:
```
[Twitter](link-to-your-twitter-profile-here)
```

Reply:
Thanks a lot ...I'll update it accordingly